modules/logging-release-notes-5-6-17.adoc

```diff
@@ -0,0 +1,50 @@
+// module included in /logging/logging-5-6-release-notes
+:_mod-docs-content-type: REFERENCE
+[id="logging-release-notes-5-6-17_{context}"]
+= Logging 5.6.17
+This release includes link:https://access.redhat.com/errata/RHSA-2024:1507[Logging Bug Fix 5.6.17]
+
+[id="logging-release-notes-5-6-17-bug-fixes"]
+== Bug fixes
+
+* Before this update, the Red Hat build pipeline didn't use the existing build details in Loki builds, leaving out information like revision, branch, and version. With this update, the Red Hat build pipeline now adds these inputs to the Loki builds, fixing the issue. (link:https://issues.redhat.com/browse/LOG-5203[LOG-5203])
+
+
+* Before this update, the configuration of the `ServiceMonitor` by Loki Operator could match many Kubernetes services, which led to Loki Operator's metrics being collected multiple times. With this update, the `ServiceMonitor` setup now only matches the dedicated metrics service. (link:https://issues.redhat.com/browse/LOG-5252[LOG-5252])
+
+
+* Before this update, the build pipeline did not include linker flags for the build date, causing Loki builds to show empty strings for `buildDate` and `goVersion`. With this update, adding the missing linker flags in the build pipeline fixes the issue. (link:https://issues.redhat.com/browse/LOG-5276[LOG-5276])
+
+* Before this update, the Loki operator ServiceMonitor in openshift-operators-redhat used static token and CA files for authentication, causing errors in the Prometheus operator in User-Workload-Monitoring on such ServiceMonitor configuration. With this update, the Loki operator ServiceMonitor in openshift-operators-redhat now references a serviceaccount token secret via a LocalReference. This approach resolves the issue, allowing the User-Workload-Monitoring Prometheus operator to handle the Loki operator ServiceMonitor successfully, enabling Prometheus to scrape the Loki operator metrics. (link:https://issues.redhat.com/browse/LOG-5242[LOG-5242])
+
+[id="logging-release-notes-5-6-17-CVEs"]
+== CVEs
+
+* link:https://access.redhat.com/security/cve/CVE-2021-35937[CVE-2021-35937]
+* link:https://access.redhat.com/security/cve/CVE-2021-35938[CVE-2021-35938]
+* link:https://access.redhat.com/security/cve/CVE-2021-35939[CVE-2021-35939]
+* link:https://access.redhat.com/security/cve/CVE-2022-3545[CVE-2022-3545]
+* link:https://access.redhat.com/security/cve/CVE-2022-41858[CVE-2022-41858]
```

Comment:
This link didn't open a CVE page properly.

Many of the others in this list don't as well. Perhaps are they just not visible, but will work properly on release date?

Reply:
Just re-checked and the link to all the CVEs work fine now. The release shipped live yesterday.